scripts/variantstore/wdl/extract/build_base.Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN apk add autoconf bash cmake g++ gcc make ninja python3-dev git openssl-dev z
 # including the numpy Python module. The main stage will then use the same base image and copy over the artifacts
 # produced by the build stage without having to install development tools or clean up after a build.
 
-ARG ARROW_VERSION=12.0.1
+ARG ARROW_VERSION=16.1.0
```

Comment:
building docker failed for me because it couldn't find ARROW 12.0.1 - so I updated it.

Reply:
uh wow that's a pretty big jump... but if the integration tests pass then fine by me. In case the integration tests do *not* pass, the "old" versions of Arrow seem to be at the "archive" location here https://archive.apache.org/dist/arrow/